src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Square square1 = new Square(5.0, Color.RED);
+        square1.draw();
+        IsoscelesTrapezoid isoscelesTrapezoid1 = new IsoscelesTrapezoid(10, 10, 20, Color.GREEN);
+        isoscelesTrapezoid1.draw();
+
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        Figure[] figures = new Figure[3];
+        figures[0] = figureSupplier.getRandomRectangle();
+        figures[1] = figureSupplier.getRandomCircle();
+        figures[2] = figureSupplier.getRandomRightTriangle();
```

Comment:
You should create several random Figures, so you will use a loop. Please don't create a `new FigureSupplier()` inside the loop. However, it seems you've correctly instantiated `FigureSupplier` before the loop. Make sure there's no instantiation inside loops elsewhere in your code.